### PR TITLE
Profile view page and linebreaks in logging

### LINKED ISSUE #36: Create a profile view of either participant
For someone who would like to view a participant, build a page under a url like `/A/<uui>` that shows the profile details, preferably not in the not-editable form view as we did in m4h since this is slightly ugly. 

It could be possible to reuse [the property definition for the config](https://github.com/match4everyone/match4everything/blob/1c7a7e371df690868789371ae9cef77e3fd8f2f0/backend/apps/matching/models/participant_info_properties.py#L6) of the information of each participant to define a type of non-editable widget for each info-field for a nice consistent style, e.g. some `get_view_widget`  method.

### PR DESCRIPTION
**Pull request by josauder**
closes #36 
Also changes logging so that it logs on multiple files. Logging on one line became obsolete with the introduction of the JSONLogFormatter, so a long time ago. Logging with linebreaks as intended works better for debugging etc.

### CODE REVIEW

File: backend/apps/matching/templates/participant/participant_info_view_form.html [hunk 1]
```diff
@@ -0,0 +1,36 @@
+{% extends 'base.html' %}
+{% load static %}
+{% load i18n %}
+
+{% block title %}
+    {% blocktrans %}
+        View Info
+    {% endblocktrans %}
```
Comment (Baschdl) [inline backend/apps/matching/templates/participant/participant_info_view_form.html:8]:
```suggestion
    {% blocktrans %}View Info{% endblocktrans %}
```

**Review by Baschdl — DISMISSED** [review]
Register page doesn't look right:
image

**Comment by feeds** [comment]
The signup in general is fixed in #48 and should be merged before

**Review by feeds — APPROVED** [review]
I just added the fields that should always be included to the private fields. Otherwise it looks good!

File: backend/locale/de/LC_MESSAGES/django.po [hunk 2]
```diff
@@ -218,6 +218,25 @@ msgstr ""
 "\n"
 "        Infos editieren"
 
+msgid "Edit Info"
+msgstr ""
```
Comment (bjrne) [inline backend/locale/de/LC_MESSAGES/django.po]: missing translation

File: backend/locale/de/LC_MESSAGES/django.po [hunk 3]
```diff
@@ -218,6 +218,25 @@ msgstr ""
...
+#| "\n"
+#| "        Edit Info\n"
+#| "    "
+msgid ""
+"\n"
+"        View Info\n"
+"    "
+msgstr ""
+"\n"
+"        Infos editieren"
```
Comment (bjrne) [inline backend/locale/de/LC_MESSAGES/django.po]: fuzzy translation

File: backend/locale/de/LC_MESSAGES/django.po [hunk 4]
```diff
@@ -218,6 +218,25 @@ msgstr ""
...
+
+msgid "View Info"
+msgstr ""
```
Comment (bjrne) [inline backend/locale/de/LC_MESSAGES/django.po]: Missing translation

**Review by bjrne — COMMENTED** [review]
LOL @feeds  :wink:

**Comment by feeds** [comment]
well...

**Pull request merged**